src/index.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import App from '@components/app';
+import App from 'app';
```

Comment:
this might be problematic, 
```suggestion
import App from './app';
```